org.olivevideoeditor.Olive.yaml

```diff
@@ -161,19 +196,44 @@ modules:
       - --enable-librsvg
       - --enable-libx264
       - --enable-libx265
+      - --enable-libmp3lame
     cleanup:
       - /share/ffmpeg/examples
     sources:
       - type: archive
         url: https://www.ffmpeg.org/releases/ffmpeg-4.2.4.tar.xz
         sha256: 0d5da81feba073ee78e0f18e0966bcaf91464ae75e18e9a0135186249e3d2a0b
 
+  - name: portaudio
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/PortAudio/portaudio/archive/refs/tags/v19.7.0.tar.gz
+        sha256: 5af29ba58bbdbb7bbcefaaecc77ec8fc413f0db6f4c4e286c40c3e1b83174fa0
+
+  - name: OpenTimelineIO
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_CXX_STANDARD=17
+      - -DOTIO_PYTHON_INSTALL=OFF
+    sources:
+      - type: git
+        url: https://github.com/AcademySoftwareFoundation/OpenTimelineIO.git
+        commit: 8cdda64f5ef924855ce503822e1aeb125e6b011e
+        tag: v0.14.1
+
   - name: olive
     buildsystem: cmake-ninja
     builddir: true
     config-opts:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DUSE_WERROR=OFF
     sources:
       - type: git
         url: https://github.com/olive-editor/olive.git
-        commit: 0699795c6f2e81d7c1594f9815dde11468e9c791
+        commit: 55d3915881b36161db6240d7e3ad7decc4730337
```

Comment:
this commit doesn't seem to match any release, and you don't provide any info

Reply:
Olive 0.2 has new nightly builds after every commit. https://www.olivevideoeditor.org/nightly.php

Reply:
but flathub isn't really for "nightlies"

Reply:
I might be wrong but i think commit 0699795c... is also just a nightly build.